neofsid/neofsid_contract.go

```diff
@@ -34,10 +34,6 @@ func _deploy(data any, isUpdate bool) {
 
 		common.CheckVersion(version)
 
-		if args[0].(bool) {
-			panic("update to non-notary mode is not supported anymore")
-		}
-
```

Comment:
I'm not sure, but do we need `switchToNotary` functions then?

Reply:
Technically you can still upgrade from pre-0.17.0.
https://github.com/nspcc-dev/neofs-contract/blob/beb9293103656cef87475bc2532388ec2501e56f/common/version.go#L10-L15